routers/cart-router.js

```diff
@@ -0,0 +1,93 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const {Cart} = require('../models/cart-model')
+const CartRouter   = express.Router()
+CartRouter.use(bodyParser.json({limit: '5000kb'}))
+const cors = require('cors')
+CartRouter.use(cors())
+CartRouter.route('/')
+.get(async (req , res) =>{
+  try{
+     const myCart = await Cart.find({})
+     res.json({myCart :myCart})
+  }catch(error){
+    res.json({status : false  , message : "couldn't collect the data" , errorMessaage : error.message})
+  }
+ 
+})
+.post(async (req , res ) =>{
+  try{
+    const data = req.body
+    const newProduct =new  Cart(data)
+    const SavedData = await newProduct.save()
+    res.json({SavedData : SavedData})
+  }
+  catch(error){
+    res.json({status : false , message : "failed to get the cart" , errorMessaage : error.message})
+  }
+})
+
+.put(async (req , res) =>{
+  const id = req.body._id
+const paramCase = req.param('case')
+  // for increment
+  if(paramCase === 'inc'){
+     try{
+      Cart.updateOne(
+    {_id :id},
+     {
+       $inc : {quantity : 1 }
+       } , 
+       
+    (err,result) =>  {
+      if (err) {
+        res.send(err);
+      } else {
+        res.json(result);
+      }
```

Comment:
I see you have passed the callback as a parameter to the updateOne function. Better would be if you use async/await with try-catch. It will simplify the code and make it look much cleaner. You could achieve the same with

    try {
        const response = await //updateOne operation;
        res.json({success: true, response});
    } catch(err) {
        res.status(500).json({success: false, errorMessage: err.message})
    }